templates/taxi/manufacturer_confirm_delete.html

```diff
@@ -0,0 +1,16 @@
+{% extends "base.html" %}
+
+{% block content %}
+	<h2>Delete manufacturer</h2>
+  <hr>
+  <p>
+    Are you sure you want to delete <strong>{{ manufacturer.name }}</strong> manufacturer?<br>
+    The data cannot be recovered.
+  </p>
+  <form action="" method="post" novalidate>
+    {% csrf_token %}
+    <input type="submit" value="Delete" class="btn btn-danger">
+  </form>
```

Comment:
Consider adding a 'Cancel' button to this form to allow users to cancel the delete operation and return to the previous page or another safe location.